conda/meta.yaml

```diff
@@ -16,14 +16,14 @@ build:
 
 requirements:
   host:
-    - python >=3.9
+    - python >=3.11,<3.14
     - pip
 
   run:
-    - python >=3.9
-    - fair-research-login >=0.2.6,<0.3.0
-    - globus-sdk >=3.0.0,<4.0.0
-    - six
+    - python >=3.11,<3.14
+    - globus-sdk=3.15.0
+    - six=1.16.0
+    - sqlite
```

Comment:
@forsyth2 Did you mean to constrain exact versions of `globus-sdk` and `six` here?

On a side-note, I don't think this file is used anymore? I'm pretty sure this file was used to host `zstash` on the `e3sm` channel, which we no longer do since adopting `conda-forge`.

I think you can just delete this file  (`meta.yaml`) if that's the case.